Assets/Scripts/Combat/CreatureStats.cs

```diff
@@ -159,31 +159,28 @@ public void PerformAttack(ObjectStats target)
         Animate("IsAttacking");
         if (!PercentRoll(HitChance())) {
             target.Animate("IsDodging");
-            DisplayPopup("Miss!");
+            DisplayPopup("Miss");
             return;
         }
         if (PercentRoll(target.DodgeChance()))
         {
             target.Animate("IsDodging");
-            target.DisplayPopup("Dodge!");
+            target.DisplayPopup("Dodge");
             return;
         }
         int dam = DamageInflicted();
         if (IsBehind(target))
         {
-            DisplayPopup("Rear attack!");
+            DisplayPopup("Rear attack");
```

Comment:
A shorter alternative could be "Flank"

Reply:
Hmm, let's see how "Backstab" looks? Flank makes it sound like it could include side attacks.